api_specs/v2/component_analyses.yaml

```diff
@@ -118,76 +304,98 @@ components:
               cvss:
                 type: string
+                example: 9.2
               is_private:
                 type: boolean
+                example: true
 
     RegisteredUser:
-      title: Vendor Private Recommendation Information
-      description: Vendor Recommendation information for Premium Response.
+      title: Vendor Private Recommendation
       type: object
       properties:
+        package:
+          type: string
+          example: "django"
+        version:
+          type: string
+          example: "1.1"
         recommendation:
           type: object
           properties:
             recommended_version:
               type: string
-            affected_version:
-              type: string
+              example: 2
             component_analyses:
               $ref: '#/components/schemas/VulnerabilityDetails'
             message:
               type: string
-            severity:
+              example: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+            highest_severity:
               type: string
+              example: "high"
             public_vulnerabilities_count:
               type: integer
+              example: 0
             private_vulnerabilities_count:
               type: integer
+              example: 0
             exploitable_vulnerabilities_count:
               type: integer
+              example: 0
             vendor_package_link:
               type: string
+              example : "https://snyk.io/vuln/maven:org.webjars%3Ajquery-form"
 
     FreeUser:
-      title: Vendor Public Recommendation Information
-      description: Vendor Recommendation information for Standard Response.
+      title: Vendor Public Recommendation
       type: object
       properties:
+        package:
+          type: string
+          example: "django"
+        version:
+          type: string
+          example: "1.1"
         recommendation:
           type: object
           properties:
             recommended_version:
               type: string
-            affected_version:
-              type: string
+              example: "3.1"
             component_analyses:
               $ref: '#/components/schemas/VulnerabilityDetails'
             message:
               type: string
-            severity:
+              example: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+            highest_severity:
               type: string
+              example: "high"
             public_vulnerabilities_count:
               type: integer
+              example: 1
             private_vulnerabilities_count:
               type: integer
+              example: 1
             registeration_link:
               type: string
+              example : "https://app.snyk.io/login"
 
     NoVulnerabilities:
       title: No Vulnerability Response
-      description: No Vulnerability Response.
       type: object
       properties:
```

Comment:
Please add package, version fields for this type as well.

Reply:
Ack